webapp/src/Entity/ImmutableExecutable.php

```diff
@@ -33,7 +37,7 @@ class ImmutableExecutable
      * @ORM\JoinColumn(name="userid", referencedColumnName="userid", onDelete="SET NULL")
```

Comment:
The `SET NULL` is still needed because we disconnect the user from the Executable?

Reply:
I don’t know. I don’t know when we filled it